includes/TypesForMembers/MemberShearPanel.js

```diff
@@ -0,0 +1,393 @@
+ * @param {Array}   member_supports_no      List of assigned member supports indexes, can be undefined
+ * @param {String}  comment                 Comment, can be undefined
+ * @param {Object}  params                  Additional parameters, can be undefined
+ */
+function MemberShearPanel (no,
+    definition_type,
+    member_supports_no,
+    comment,
+    params) {
+    if (arguments.length > 0) {
+        this.member_shear_panel = createMemberShearPanel(no, definition_type, member_supports_no, comment, params);
+    }
+}
+
+/**
+ * Creates Trapezoidal sheeting Member Shear Panel
+ * @param {Number}  no                          Index of Member Shear Panel, can be undefined
+ * @param {Array}   member_supports_no          List of assigned member supports indexes, can be undefined
+ * @param {String}  position_on_section         Position on section (UPPER_FLANGE, CENTROID, LOWER_FLANGE, DEFINE), can be undefined (UPPER_FLANGE as default)
+ * @param {String}  trapezoidal_sheeting_name   Trapezoidal sheeting name
+ * @param {String}  fastening_arrangement       Fastening arrangement (EVERY_RIB, EVERY_SECOND_RIB), can be undefined (EVERY_RIB as default)
+ * @param {String}  comment                     Comment, can be undefined
+ * @param {Object}  params                      Additional parameters, can be undefined
+ */
+MemberShearPanel.prototype.TrapezoidalSheeting = function (no,
+    member_supports_no,
+    position_on_section,
+    trapezoidal_sheeting_name,
+    fastening_arrangement,
+    comment,
+    params) {
+    this.member_shear_panel = createMemberShearPanel(no, "TRAPEZOIDAL_SHEETING", member_supports_no, comment, params);
+    ASSERT(typeof trapezoidal_sheeting_name !== "undefined", "Trapezoidal sheeting name must be defined");
+    this.member_shear_panel.sheeting_name = trapezoidal_sheeting_name;
+    this.member_shear_panel.position_on_section = GetMemberShearPanelPositionOnSection(position_on_section);
+    this.member_shear_panel.fastening_arrangement = GetMemberShearPanelFasteningArrangement(fastening_arrangement);
+};
+
+/**
+ * Creates Bracing Member Shear Panel
+ * @param {Number}  no                      Index of Member Shear Panel, can be undefined
+ * @param {Array}   member_supports_no      List of assigned member supports indexes, can be undefined
+ * @param {String}  position_on_section     Position on section (UPPER_FLANGE, CENTROID, LOWER_FLANGE, DEFINE), can be undefined (UPPER_FLANGE as default)
+ * @param {String}  diagonals_section_name  Diagonal section name
+ * @param {String}  posts_section_name      Posts section name
+ * @param {String}  comment                 Comment, can be undefined
+ * @param {Object}  params                  Additional parameters
+ */
+MemberShearPanel.prototype.Bracing = function (no,
+    member_supports_no,
+    position_on_section,
```

Comment:
Add material otherwise invalid object is created

Reply:
Where should be material added? Member shear panel has no material, as I can see in its API.